catalog/spatial/registry/registry-federation-admin-service-impl/pom.xml

```diff
@@ -93,6 +93,13 @@
             <artifactId>filter-proxy</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <!-- Upgrade mockito to version 2.1.0 for this module to get final mock capability -->
```

Comment:
Should this comment be here?

Reply:
Looks like a copy and paste from a project where I had to use the newer version of Mockito for the listed reasons and I couldn't take it on to upgrade Mockito for all.

Reply:
❓ Is there a reason we're not using a version property in the root pom or even dependency management on this?

Reply:
just because we are not yet ready to upgrade Mockito for all projects. The current version used by everybody doesn't support mocking final methods.

Reply:
To follow up, I believe doing so would would require many more fixes throughout, which seems out of the scope and unnecessary at this point. Please correct me if I am wrong.

Reply:
Correct. Upgrade of Mockito is already on the test cop team's radar. For now, make sure that if you are upgrading a project to use the latest version of Mockito then all that project should be updated. As such, I am not a fan of using `MockitoJUnitRunner.Silent.class`